.github/workflows/php-cs-fixer.yml

```diff
@@ -11,7 +11,7 @@ jobs:
       - name: PHP-CS-Fixer
         uses: docker://oskarstark/php-cs-fixer-ga
         with:
-          args: --config=.php-cs-fixer.dis --allow-risky=yes
+          args: --config=.php-cs-fixer.dist.php --allow-risky=yes
```

Comment:
```suggestion
          args: --config=.php-cs-fixer.dist --allow-risky=yes
```

Reply:
@shuvroroy Can you please remove the `.php` extension? And change the name of the related file.